scripts/run_commands

```diff
@@ -30,8 +32,11 @@ conda config --add channels conda-forge
 conda update --all --yes
 conda clean -tipy
 
+# Install common Python tools.
+conda install --yes --quiet python="${pyver}" pip setuptools
```

Comment:
This seems like it would slow things down by being another long solve.  Can you just combine this with the command below?